rcode-reader/src/main/java/eu/rssw/pct/elements/TableElement.java

```diff
@@ -0,0 +1,78 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.AccessibleClassElement;
+import eu.rssw.pct.RCodeInfo;
+
+public class TableElement extends AccessibleClassElement {
+  protected int flags;
+  protected int crc;
+  protected int prvte;
+  protected VariableElement[] fields;
+  protected IndexElement[] indexes;
+  private String beforeTableName;
+
+  public TableElement(String name, Set<AccessType> accessType, byte[] segment, int currentPos, int textAreaOffset,
+      ByteOrder order) {
+    super(name, accessType);
+
+    int fieldCount = ByteBuffer.wrap(segment, currentPos, Short.BYTES).order(ByteOrder.LITTLE_ENDIAN).getShort();
+    int indexCount = ByteBuffer.wrap(segment, currentPos + 2, Short.BYTES).order(ByteOrder.LITTLE_ENDIAN).getShort();
+    this.flags = ByteBuffer.wrap(segment, currentPos + 4, Short.BYTES).order(ByteOrder.LITTLE_ENDIAN).getShort();
+    this.crc = ByteBuffer.wrap(segment, currentPos + 6, Short.BYTES).order(ByteOrder.LITTLE_ENDIAN).getShort();
+    this.prvte = ByteBuffer.wrap(segment, currentPos + 8, Short.BYTES).order(ByteOrder.LITTLE_ENDIAN).getShort();
+    int indexComponentCount = ByteBuffer.wrap(segment, currentPos + 10, Short.BYTES).order(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "indexComponentCount" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1481)